domains/trix.json

```
@@ -0,0 +1,13 @@
+{
+    "owner": {
+        "username": "tr1xem",
+        "email": ["kmtechnical2006@gmail.com"]
```

Making it an array isn't necessary.
```suggestion
        "email": "kmtechnical2006@gmail.com"
```